backend/validators/auth.validator.js

```diff
@@ -16,12 +17,13 @@ const signUpSchema = async (req, res, next) => {
   validateRequest(req, res, next, schema, "body");
 };
 
-const signInSchema = async (req, res, next) => {
+// schema of user signin api in request body for validation using joi
+const userSignInSchema = async (req, res, next) => {
   const schema = Joi.object({
     email: Joi.string().email().lowercase().required(),
```

Comment:
why it is declared again when we can use single declared email validation, check on this